index.php

```diff
@@ -14,15 +15,20 @@
 $key = strval( getenv( 'KEY' ) );
 
 $auth = new APNSCredentials( $key_id, $team_id, $key );
-$configuration = APNSConfiguration::production( $auth );
+$configuration = APNSConfiguration::sandbox( $auth );
 $configuration->setUserAgent( strval( getenv( 'USER_AGENT' ) ) );
-$client = new APNSClient( $configuration );
-$client->setDebug( true );
+$client = new APNSClient( $configuration, new CurlMultiplexedNetworkService() );
+$client->setDebug( false );
 
 echo "\t Connected.\n";
+$title = 'Title';
+$message = 'Timestamp ' . time();
+echo "\t Will send notification with:\n";
+echo "\t - Title: " . $title . PHP_EOL;
+echo "\t - Message: " . $message . PHP_EOL;
```

Comment:
This just made it easier to verify that the last notification set I received was the same that was sent from here.